assets/js/src/segments/list.jsx

```diff
@@ -263,6 +270,16 @@ class SegmentList extends React.Component {
         <td data-colname={MailPoet.I18n.t('description')}>
           <abbr>{ segment.description }</abbr>
         </td>
+        { (mailpoetTrackingEnabled === true) ? (
```

Comment:
Just a detail but it caught my eye. Why can't use just `if (mailpoetTrackingEnabled)` as we did on the line 28 (`display: mailpoetTrackingEnabled`) ?